pkg/dashboard/ui/src/app/components/projects/projects.component.js

```diff
@@ -169,7 +170,7 @@
                     if (lodash.isNonEmpty(messagesArray) || lodash.isNonEmpty(notEmptyMessagesArray)) {
                         var messages = formatMessage(messagesArray);
                         var notEmptyMessages = formatMessage(notEmptyMessagesArray);
-                        var promise = lodash.isEmpty(messages) ? $q.resolve() : DialogsService.alert(messages);
+                        var promise = lodash.isEmpty(messages) ? $q.when() : DialogsService.alert(messages);
```

Comment:
Oh, sorry, I deleted this comment a minute after I created it because I realized that I was wrong and `$q.resolve` _does_ truly exist. So I deleted the comment because you didn't have to change it.
However if you already changed it, it's okay to leave it like that, no need to change back to `$q.resolve`.